pom.xml

```diff
@@ -133,21 +133,23 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <project.docs.version>${project.version}</project.docs.version>
-    <java.version>11</java.version>
-    <failsafe.version>2.22.2</failsafe.version>
-    <surefire.version>2.22.2</surefire.version>
+    <java.version>17</java.version>
+
     <maven.version>3.8.4</maven.version>
-    <maven.compiler.version>3.10.1</maven.compiler.version>
+    <maven.compiler.version>3.11.0</maven.compiler.version>
     <maven.compiler.release>${java.version}</maven.compiler.release>
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
 
-    <activemq.version>5.17.3</activemq.version>
+    <failsafe.version>2.22.2</failsafe.version>
+    <surefire.version>2.22.2</surefire.version>
```

Comment:
you're not liking the (stable!) 3M-releases here, but use it for camel? 😉 I don't really mind it, but it feels kinda strange.. anyway, failsafe and surefire 3 are out for like a week or so. we should upgrade this in a follow up PR.